package-lock.json

```diff
@@ -1,2639 +0,0 @@
-{
-  "name": "postgis-preview",
```

Comment:
Why are we ditching package-lock?

Reply:
We need to go with yarn or npm, our team's been going the yarn route lately.  Do you have a preference?